multipaz-barcodes/src/androidMain/kotlin/org/multipaz/barcodes/BarcodeScanner.android.kt

```diff
@@ -0,0 +1,57 @@
+package org.multipaz.barcodes
+
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Rect
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asAndroidBitmap
+import com.google.android.gms.tasks.Tasks
+import com.google.mlkit.vision.barcode.common.Barcode as MLBarcode
+import com.google.mlkit.vision.barcode.BarcodeScannerOptions
+import com.google.mlkit.vision.barcode.BarcodeScanning
+
+import com.google.mlkit.vision.common.InputImage
+
+actual fun scanBarcode(image: ImageBitmap): List<Barcode> {
+
+    val inputImage = InputImage.fromBitmap(
+        /* bitmap = */ image.asAndroidBitmap(),
```

Comment:
Any reason the names are commented out? Can't we just do:
    bitmap = image.asAndroidBitmap(),
    rotationDegrees = 0)

?

Reply:
Yes, it's because `InputImage` is declared in a Java library so we can't use Kotlin-isms like that.